mslice/widgets/workspacemanager/workspacemanager.py

```diff
@@ -118,7 +118,8 @@ def add_workspace_dialog(self):
         dialog.setLabelText("Choose a workspace to add:")
         dialog.setOptions(QInputDialog.UseListViewForComboBoxItems)
         dialog.setComboBoxItems(items)
-        dialog.exec_()
+        if dialog.exec_() != QInputDialog.Accepted:
+            raise RuntimeError('dialog cancelled')
```

Comment:
Would it be slightly better to `return None` instead of raising an error, because cancelling a dialog isn't a runtime error?

Reply:
I agree! I thought about this vs. following the style that was already used in the file, but returns are just nicer